ops/docker/Dockerfile.init

```diff
@@ -0,0 +1,31 @@
+FROM node:20
```

Comment:
Why `node:20`? Seems like we should possibly be simply pulling a Cargo build env?  If we need Node at runtime for the ts scripts, perhaps we would be better off copying the built binary into a Node env? Presumably we do not need to build any rust at runtime?

Reply:
good point! will change that